moveit_demo_nodes/run_moveit_cpp/config/panda.ros2_control.xacro

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
```

Comment:
Interesting, is this required for simulated robots and could this be generated from inline tags in the description?

Reply:
This file is required by all robots running on ros2_control. The generation for a robot is not so simple. This has to be actually provided with the robot's driver.

You could generate some general descriptions from the joints in URDF and assume they all have can receive position commands and provide position feedback. I have already some templates for this. This will work perfectly with the MoveIt --> Joint Trajectory Controller --> Fake System. Nevertheless, when using real hardware those interfaces have to correspond to the driver.

If you have any questions regarding the new ros2_control Architecture or FakeRobot (the same as the FakeSystem) just ping me :)